eng/tools/publish-to-npm.ps1

```diff
@@ -157,7 +157,7 @@ try {
             }
         }
         elseif ($addTag -and $publishToNpm) {
-            $nameAndVersion = "$($p.Project.name)@$($p.Project)"
+            $nameAndVersion = $p.Project.name + "@" + $p.Project.version
```

Comment:
is the .version here the only real change?

Reply:
within this line, yes. for the pr, no. also having npm token set as env var and passing in tags above